docfx-extensions/Intent.DocFx.TagExtensions/Intent.DocFx.TagExtensions.csproj

```diff
@@ -0,0 +1,78 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Let's rather make this an SDK style `.csproj` file, it's just so much nicer to work with, even works for .NET Framework projects.

Reply:
I'll convert and see what happens